quixstreams/models/topics/manager.py

```diff
@@ -0,0 +1,365 @@
+              its "auto_create_topics" option.
+
+        :return: TopicConfig or None, depending on function arguments
+        """
+        if topic_config:
+            return self._topic_config_with_defaults(
+                num_partitions=topic_config.num_partitions,
+                replication_factor=topic_config.replication_factor,
+                extra_config=topic_config.extra_config,
+                extra_config_defaults=extra_config_defaults,
+            )
+        else:
+            if not auto_create_config:
+                return
+            return self._topic_config_with_defaults(
+                extra_config_defaults=extra_config_defaults,
+            )
+
+    def topic(
+        self,
+        name: str,
+        value_deserializer: Optional[DeserializerType] = None,
+        key_deserializer: Optional[DeserializerType] = "bytes",
+        value_serializer: Optional[SerializerType] = None,
+        key_serializer: Optional[SerializerType] = "bytes",
+        config: Optional[TopicConfig] = None,
+        auto_create_config: bool = True,
+        timestamp_extractor: Optional[TimestampExtractor] = None,
+    ) -> Topic:
+        """
+        A convenience method for generating a `Topic`. Will use default config options
+        as dictated by the TopicManager.
+
+        :param name: topic name
+        :param value_deserializer: a deserializer type for values
+        :param key_deserializer: a deserializer type for keys
+        :param value_serializer: a serializer type for values
+        :param key_serializer: a serializer type for keys
+        :param config: optional topic configurations (for creation/validation)
+        :param auto_create_config: if no "topic_config", create one; Default - True
+            > NOTE: this setting is generally manipulated by the Application class via
+              its "auto_create_topics" option.
+        :param timestamp_extractor: a callable that returns a timestamp in
+            milliseconds from a deserialized message.
+
+        :return: Topic object with creation configs
+        """
+        name = self._apply_topic_prefix(name)
+        if len(name) > self._max_topic_name_len:
+            raise TopicNameLengthExceeded(
+                f"Topic {name} exceeds the {self._max_topic_name_len} character limit"
+            )
+        topic = Topic(
+            name=name,
+            value_serializer=value_serializer,
+            value_deserializer=value_deserializer,
+            key_serializer=key_serializer,
+            key_deserializer=key_deserializer,
+            config=self._process_topic_configs(
+                config, auto_create_config=auto_create_config
+            ),
+            timestamp_extractor=timestamp_extractor,
+        )
+        self._topics[name] = topic
+        return topic
+
+    def changelog_topic(
+        self,
+        topic_name: str,
+        suffix: str,
```

Comment:
Why do we name it `suffix` instead of `store_name`?

The docstring says it's a store name:
> :param suffix: name of storage type (default, rolling10s, etc.)